requirements.txt

```diff
@@ -6,3 +6,4 @@ psycopg2-binary==2.7.7
 pytz==2018.9
 whitenoise==3.2.2
 django-spa==0.2.0
+djangorestframework-jsonapi
```

Comment:
Can we specify a version? It wouldn’t be the first time I had a library update automatically on me and break something.